deeplearning4j-scaleout/deeplearning4j-scaleout-parallelwrapper/src/main/java/org/deeplearning4j/parallelism/trainer/DefaultTrainer.java

```diff
@@ -187,12 +189,19 @@ public void shutdown() {
     }
 
     protected void fit(DataSet dataSet) {
-        if (replicatedModel instanceof MultiLayerNetwork) {
-            ((MultiLayerNetwork) replicatedModel).setLastEtlTime(lastEtlTime.get());
-            ((MultiLayerNetwork) replicatedModel).fit(dataSet);
-        } else if (replicatedModel instanceof ComputationGraph) {
-            ((ComputationGraph) replicatedModel).setLastEtlTime(lastEtlTime.get());
-            ((ComputationGraph) replicatedModel).fit(dataSet);
+        try {
+            if (replicatedModel instanceof MultiLayerNetwork) {
+                if (lastEtlTime == null)
+                    lastEtlTime = new AtomicLong(0);
```

Comment:
Different handling for lastEtlTime == null for ComputationGraph and MultiLayerNetwork?